chef/cookbooks/chef_handler/attributes/default.rb

```diff
@@ -20,7 +20,7 @@
 
 default["chef_handler"]["root_user"] = "root"
 
-case platform
+case platform_family
```

Comment:
I'm not sure. I think, the ``mac_os_x_server`` is a platform, but likely not platform family